server/src/main/java/org/opensearch/rest/action/cat/RestTable.java

```diff
@@ -64,13 +64,20 @@ public class RestTable {
 
     public static RestResponse buildResponse(Table table, RestChannel channel) throws Exception {
         RestRequest request = channel.request();
-        XContentType xContentType = XContentType.fromMediaTypeOrFormat(request.param("format", request.header("Accept")));
+        XContentType xContentType = getxContentType(request);
         if (xContentType != null) {
             return buildXContentBuilder(table, channel);
         }
         return buildTextPlainResponse(table, channel);
     }
 
+    private static XContentType getxContentType(RestRequest request) {
```

Comment:
Nitpick, may be `getXContentType` would be more readable

Reply:
agree :+1:  fat fingers..